.buildkite/pipeline.yaml

```diff
@@ -11,7 +13,7 @@ steps:
       mv buildifier /usr/local/bin/
 
       # Bazel handles the C++ toolchain
-      bazel build @llvm_toolchain//:clang-format
+      bazel build @llvm_toolchain//:clang-format --remote_cache=$$CI_BAZEL_REMOTE_CACHE --google_credentials=/mnt/gcloud-service-account/gcloud-service-account.json
```

Comment:
It looks like this `$$` is needed because of the two-level interpolation going on in Buildkite. https://buildkite.com/docs/agent/v3/cli-pipeline#environment-variable-substitution